test/unit/org/apache/cassandra/utils/memory/NativeAllocatorTest.java

```diff
@@ -22,105 +22,140 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 import com.google.common.util.concurrent.Uninterruptibles;
+import org.junit.Assert;
+import org.junit.Before;
 import org.junit.Test;
 
-import junit.framework.Assert;
 import org.apache.cassandra.utils.concurrent.OpOrder;
 
 public class NativeAllocatorTest
 {
+    ScheduledExecutorService exec;
+    OpOrder order;
+    OpOrder.Group group;
+    CountDownLatch canClean;
+    CountDownLatch isClean;
+    AtomicReference<NativeAllocator> allocatorRef;
+    AtomicReference<OpOrder.Barrier> barrier;
+    NativePool pool;
+    NativeAllocator allocator;
+    Runnable markBlocking;
```

Comment:
Nit: could be private
```suggestion
    private ScheduledExecutorService exec;
    private OpOrder order;
    private OpOrder.Group group;
    private CountDownLatch canClean;
    private CountDownLatch isClean;
    private AtomicReference<NativeAllocator> allocatorRef;
    private AtomicReference<OpOrder.Barrier> barrier;
    private NativePool pool;
    private NativeAllocator allocator;
    private Runnable markBlocking;
```